rir/src/runtime/Function.h

```diff
@@ -47,21 +47,22 @@ typedef SEXP FunctionSEXP;
 #pragma pack(push)
 #pragma pack(1)
 struct Function {
+public:
```

Comment:
indentation seems off, maybe your clang format somehow failed to run?

Reply:
yes, it failed :) `<q><n><enter>`
sorry, will start to use it